OCPBUGS#14864: Add known issue for Selinux relabeling issue for 4.13 (#61561)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

* {product-title} 4.12 clusters on the Nutanix platform may have an `Upgradeable=False` condition if they are missing configuration needed for the Nutanix Cloud Control Manager (CCM). To resolve this condition, see: link:https://access.redhat.com/solutions/7014068[How to create the ConfigMaps and Secrets needed to upgrade to OpenShift 4.13 when using Nutanix as a Platform].

> * Currently, when using a persistent volume (PV) that contains a very high file count, the pod may not start or takes an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])

Reviewer: ISG: Use "may" only to indicate permissions. 

```suggestion
* Currently, when using a persistent volume (PV) that contains a very high file count, the pod might not start or can an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```

Reviewer 2: same follow up comment as - https://github.com/openshift/openshift-docs/pull/61570#discussion_r1242490942

Author: Done.

> * Currently, when using a persistent volume (PV) that contains a very large number of files, the pod might not start or can take an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. Also, for ODF users, see this link:https://access.redhat.com/solutions/6906261[workaround article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])

Reviewer 3: The ODF known issues does not container the KB#6906261 so please remove in order to be aligned with their release notes. Thanks!

Author: Done.
